src/wacks4shop/wack.py

```diff
@@ -82,9 +85,16 @@ def main(db: Wackabase, dry=False):
 
     args = parser.parse_args()
 
-    # create the database folder if it doesn't exit
-    Path(DATABASE_DIR).mkdir(parents=True, exist_ok=True)
-
-    wackabase = Wackabase(DATABASE_DIR)
-    main(db=wackabase, dry=args.dry)
-    wackabase.write_success()
+    # grab the lock
+    try:
+        lock = FileLock(LOCKFILE, timeout=5)
+        with lock:
+            sleep(30)
```

Comment:
why do we need to sleep here? won't this make each run really slow?

Reply:
ooooooooooops that was for when I was testing